### PR TITLE
simplify pull push

### PR DESCRIPTION
**Pull request by oetiker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -920,10 +920,8 @@ git remote -v
 To pull all the data from the remote into the local repository. Use the `fetch` and `merge` subcommands. Before doing this, check that the current branch is set to `master`.
```
Comment (pbdigital-dot-org) [inline README.md:920]: This should then be changed to omit the fetch/merge commands, to something like:
"To update the remote into the local repository. Use the `pull` subcommand. Before doing this, make certain the current branch is set to `master`."

File: README.md [hunk 2]
```diff
@@ -920,10 +920,8 @@ git remote -v
...
 
 ```none
-git branch
 git checkout master
-git fetch upstream
-git merge upstream/master
+git pull upstream master
 ```
 
 **git fetch** only downloads new data from a remote repository - but it doesn't integrate any of this new data into the working files.
```
Comment (pbdigital-dot-org) [inline README.md:927]: This and the following **git merge** description can be deleted.

Typos: In "The OmniOS Extra Build System" section intro "...however, if certains details have not been..." s/certains/certain/ & also in "Structure" section s/coose/choose/

File: README.md (hunk 1 above)
Comment (citrus-it) [reply README.md:920]: I think this is largely a matter of preference. I tend to use `git fetch` and `git rebase`. If we're going to suggest `git pull`, I suggest adding `--rebase`.

File: README.md (hunk 2 above)
Comment (pbdigital-dot-org) [reply README.md:927]: I have just noticed, the beginning of the template `build.sh` places the "shbang" in the code fold. 
I can update this and the typos, also the "simplify pull push". if you let me know what has been decided with simplifying the pull push section. 
One other thing, should the words "CDDL HEADER" be in the code fold or not?

Comment (citrus-it) [reply README.md:927]: Oh yes, I missed that.

The fold should look like this:

```
#!/usr/bin/bash
#
# {{{ CDDL HEADER
#
# This file and its contents are supplied under the terms of the
# Common Development and Distribution License ("CDDL"), version 1.0.
# You may only use this file in accordance with the terms of version
# 1.0 of the CDDL.
#
# A full copy of the text of the CDDL should have accompanied this
# source. A copy of the CDDL is also available via the Internet at
# http://www.illumos.org/license/CDDL.
# }}}
```

Then when you open in vim, you get:

```
#!/usr/bin/bash
#
+-- 11 lines: # CDDL HEADER-----------------------------------------------------

# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
```

**Pull request closed without merging**